client/db/bolt/db.go

```diff
@@ -308,6 +308,13 @@ func (db *BoltDB) CreateAccount(ai *dexdb.AccountInfo) error {
 	})
 }
 
+// UpdateAccountKey Updates account's EncKey.
+func (db *BoltDB) UpdateAccountKey(ai *dexdb.AccountInfo) error {
+	return db.acctsUpdate(func(accts *bbolt.Bucket) error {
+		return accts.Put(ai.EncKey, ai.Encode())
```

Comment:
Non-disabled accounts are keyed on `ai.Host` I believe (See `(*BoltDB).{CreateAccount,AccountPaid}`).  I think the implementation of `UpdateAccount` should look similarly to `AccountPaid`, except instead of `acct.Put(feeProofKey, proof.Encode())`, we'd be updating the account info like `acct.Put(accountKey, ai.Encode())`.

I suppose the valid sub-buckets and keys inside each bucket could be documented better though since this isn't totally clear without studying the code.